README.md

```diff
@@ -7,8 +7,8 @@ ocaml-hex -- a minimal library providing hexadecimal converters
 ```ocaml
 #require "hex";;
 # Hex.of_string "Hello world!";;
-- : Hex.t = "48656c6c6f20776f726c6421"
```

Comment:
for me on 1.4.0, this is still valid:
```
# #require "hex";;
# Hex.of_string "Hello world!";;
- : Hex.t = 48656c6c6f20776f726c6421
```

Reply:
A `'Hex` constructor is needed.

Reply:
but it is not printed in utop (at least on my machine)

Reply:
That's very strange!